lib/iex/lib/iex/helpers.ex

```diff
@@ -325,9 +325,26 @@ defmodule IEx.Helpers do
 
   @doc """
   Retrieves the nth expression's value from the history.
+  Where nth is iex(nth)>
 
-  Use negative values to look up expression values relative to the current one.
-  For instance, v(-1) returns the result of the last evaluated expression.
+  It is usefull to bound expression on the history to a variable
+
+  ## Examples
+
+      iex(1)> 40 + 2
+      42
+      iex(2)> [1, 2, third | rest] = [1, 2, 3, 4, 6, 7, 8, 9]
+      [1, 2, 3, 4, 6, 7, 8, 9]
+      iex(3)> third
+      3
+      iex(4)> rest
+      [4, 6, 7, 8, 9]
+      iex(5)> v(2)
+      [1, 2, 3, 4, 6, 7, 8, 9]
+      iex(6)> v(1)
+      42
+      iex(7)> meanning_of_life = v
+      42
```

Comment:
I think this addition breaks the flow of the docs and make them really hard to understand. For example, "It is useful to bound an expression on the history to a variable" adds nothing to the docs of `v/1`, which actually is there exactly so that you don't have to bound expressions to variables in IEx. Also, the example (which I agree could be useful) just show `v/1` with positive numbers, but it would be useful to see it used with negative numbers as well. Wdyt? :)